finder-app/manual-linux.sh

```diff
@@ -35,9 +37,18 @@ if [ ! -e ${OUTDIR}/linux-stable/arch/${ARCH}/boot/Image ]; then
     git checkout ${KERNEL_VERSION}
 
     # TODO: Add your kernel build steps here
+    sudo apt-get update
+    sudo apt-get install -y --no-install-recommends bc u-boot-tools kmod cpio flex bison libssl-dev psmisc
+    sudo apt-get install -y qemu-system-arm
```

Comment:
@thra6614 these steps don't necessarily need to be a part of your scripts, they can be a requirement of the host setup.  As written these assume an ubuntu host running the scripts.  Typically the user will do application installs outside a script like this, so they have more control over the process.